e2e/tests/index.d.ts

```diff
@@ -3,6 +3,7 @@
 declare namespace Cypress {
     interface Chainable {
         addFlight(flight: object): Chainable<Element>;
+        addTestFlight(flight: object): Chainable<Element>;
```

Comment:
should we name this "addNonPassengerFlight"

Reply:
This is to add flight it can be passenger or non passenger depending upon content .